packages/jest-types/README.md

```diff
@@ -0,0 +1,7 @@
+# @jest/types
+
+This package contains shared types of Jest's packages.
+
+If you're a user of Jest and looking for types, you're probably looking for [@types/jest](https://www.npmjs.com/package/@types/jest).
```

Comment:
Thanks. I think it would be better to suggested importing from `@jest/globals`. This is Jest's own package. Hence imported types will be always up to date and in better shape than the ones from `@types/jest`.

Reply:
Sorry I haven't used `@jest/globals` before. Does this mean people need to explicitly import `describe` and the likes?


Like so:

```typescript
import { describe, it, expect } from '@jest/globals';

describe('my tests', () => {
  it('works', () => {
    expect(1).toBe(1);
  });
});
```


I personally prefer `@types/jest`, because it means quite a bit less importing. So perhaps just mention it like @SimenB  suggested?

If you want me to remove the link to `@types/jest`, I'd be happy to do that though :) let me know

Reply:
We should mention both, but mention `@jest/globals` first as that's the thing that's maintained in this repo

Reply:
A agreed. Both can be mentioned, of course

Reply:
updated :)